.drone.yml

```diff
@@ -150,13 +151,14 @@ steps:
       mount:
         - .
 
-  - name: clone
+  - name: checkout
```

Comment:
Changed to match name in unit-test pipeline (forgot to rename this one in previous PR)